filters/builtin/origin_marker.go

```diff
@@ -0,0 +1,78 @@
+package builtin
+
+import (
+	"time"
+
+	"github.com/zalando/skipper/eskip"
+	"github.com/zalando/skipper/filters"
+)
+
+const (
+	OriginMarkerName = "originMarker"
+)
+
+type originMarkerSpec struct{}
+
+// OriginMarker carries information about the origin of a route
```

Comment:
@aryszka I'm not sure if we should also document it in filters.md - since it's a rather internal thing. what do you think?